srcpkgs/elvish/template

```diff
@@ -11,8 +11,9 @@ license="BSD-2-Clause"
 homepage="https://elv.sh/"
 changelog="https://elv.sh/blog/${version}-release-notes.html"
 distfiles="https://github.com/elves/elvish/archive/v${version}.tar.gz"
-checksum=6a6006015f44def98676eaed611702b000d66838c0e76da572d517d9bde5c388
+checksum=3a4b93c3c99fe2f9847de35d64be24e2d4b9c12d429cd9831b4571993a66bb7a
 register_shell="/bin/elvish"
+make_check=no # issues with go unit tests when running via xbps-src
```

Comment:
the only tests that fail are something that seems to depend on some file structure, so just `make_check_args="-skip TestLocation"`

Reply:
Agree, that is the better solution. I was not aware of the `-skip` parameter for `go test`.